impress-org/givewp-next-gen
# Chore: update all js packages and dependencies

## Description



- Updates all JS packages and dependencies 
- uses new react 18 createRoot in form builder
- removes unnecessary react imports with automatic jsx runtime
- updates to existing styles to account for package updates
- improved hiding wp elements when viewing a form
- added notice popup when form has been updated
- lock initial sections by default when creating a form

## Affects



The build process & various styles in the builder.

## Visuals

<img width="1440" alt="Screenshot 2023-01-23 at 5 06 28 PM" src="https://user-images.githubusercontent.com/10138447/214160601-356e1342-14dd-4ffb-8336-13f5d6f849f9.png">


## Testing Instructions



- Pull this down, run `npm install && npm run build`
- Make sure the form builder loads & nothing looks out of place
- Make sure a form renders on the front-end

## Pre-review Checklist
